src/traffic_server/traffic_server.cc

```diff
@@ -2030,13 +2034,26 @@ main(int /* argc ATS_UNUSED */, const char **argv)
         proxyServerCheck.wait(lock, [] { return et_net_threads_ready; });
       }
 
+      {
+        std::unique_lock<std::mutex> lock(pluginInitMutex);
+        plugin_init_done = true;
+        lock.unlock();
+        pluginInitCheck.notify_one();
```

Comment:
Any good reason to not notify all? It might be the case in the future that other things wait on plugin initialization, although really this could be fixed at that point.

Reply:
Actually I thought about changing it to `notify_all()`, but then I thought since we only have one place that's waiting for it, probably should not change it.